modelci/cli/modelhub.py

```diff
@@ -14,54 +14,39 @@
 from pathlib import Path
 from typing import Dict, List, Optional
 
-import click
 import requests
 import typer
 import yaml
 from pydantic import ValidationError
 
 from modelci.app import SERVER_HOST, SERVER_PORT
 from modelci.hub.init_data import export_model
-from modelci.hub.manager import register_model_from_yaml
 from modelci.hub.publish import _download_model_from_url
 from modelci.types.models import Framework, Engine, IOShape, Task, Metric
-from modelci.types.models.mlmodel import MLModelInYaml
+from modelci.types.models.mlmodel import MLModelInYaml, MLModel
```

Comment:
`types.models`

`type.models.mlmodel`

`MLModel`, `MLModelIn`

a little confusing and hard to maintain

Reply:
`types.models` is the package of all data class definitions (from MVC). It is confusing because the word `model` from MVC is similar to the `model` from ML.

I would rather keep the word `model` from MVC as all other python backends use this word (Django, FastAPI, Flask, etc.) and change the word from ML to another.

Reply:
If you have any proposal to remove the ambiguity.